rebasebot/bot.py

```diff
@@ -439,14 +439,17 @@ def _is_push_required(gitwd: git.Repo, dest: GitHubBranch, source: GitHubBranch,
     return True
 
 
-def _is_pr_available(dest_repo: Repository, rebase: GitHubBranch) -> Tuple[ShortPullRequest, bool]:
+def _is_pr_available(dest_repo: Repository, dest: GitHubBranch, rebase: GitHubBranch) -> Tuple[ShortPullRequest, bool]:
     logging.info("Checking for existing pull request")
-    try:
-        gh_pr = dest_repo.pull_requests(head=f"{rebase.ns}:{rebase.branch}").next()
-        logging.info("Found existing pull request: %s", gh_pr.html_url)
-        return gh_pr, True
-    except StopIteration:
-        pass
+
+    pull_requests = dest_repo.pull_requests(base=dest.branch)
```

Comment:
do we no longer need to catch the `StopIteration` exception?

Reply:
No, instead of calling .next() that trows StopIteration, we for loop over the slice.